spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -1510,9 +1519,11 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
                     batchSize: Int,
                     trigger: Trigger,
                     valRdd: JavaRDD[Sample],
-                    vMethods: JList[String]): Unit = {
+                    vMethods: JList[String],
+                    criterion: String = null,
```

Comment:
cc @dding3  https://github.com/intel-analytics/BigDL/pull/1149 has implemented it. Please update accordingly.